extensions/wikia/AnalyticsEngine/js/universal_analytics.js

```diff
@@ -0,0 +1,437 @@
+    window.ga('set', 'dimension19', window.wgArticleType);                          // ArticleType
+    /*
+     * Remove when SOC-217 ABTest is finished
+     */
+    window.ga('set', 'dimension20', getUnconfirmedEmailUserType());                 // UnconfirmedEmailUserType
+    /*
+     * end remove
+     */
+
+    /**** Include A/B testing status ****/
+    if (window.Wikia && window.Wikia.AbTest) {
+        var abList = window.Wikia.AbTest.getExperiments( /* includeAll */ true),
+            abExp, abGroupName, abSlot, abIndex,
+            abForceTrackOnLoad = false,
+            abCustomVarsForAds = [];
+        for (abIndex = 0; abIndex < abList.length; abIndex++) {
+            abExp = abList[abIndex];
+            if (!abExp || !abExp.flags) {
+                continue;
+            }
+            if (!abExp.flags.ga_tracking) {
+                continue;
+            }
+            if (abExp.flags.forced_ga_tracking_on_load && abExp.group) {
+                abForceTrackOnLoad = true;
+            }
+            abSlot = window.Wikia.AbTest.getGASlot(abExp.name);
+            if (abSlot >= 40 && abSlot <= 49) {
+                abGroupName = abExp.group ? abExp.group.name : (abList.nouuid ? 'NOBEACON' : 'CONTROL');
+                _gaWikiaPush(['set', 'dimension' + abSlot, abGroupName]);
+                abCustomVarsForAds.push(['ads.set', 'dimension' + abSlot, abGroupName]);
+            }
+        }
+        if (abForceTrackOnLoad) {
+            var abRenderStart = window.wgNow || (new Date()), abOnLoadHandler;
+
+            abOnLoadHandler = function () {
+                var renderTime = (new Date()).getTime() - abRenderStart.getTime();
+                setTimeout(function () {
+                    window.guaTrackEvent('ABtest', 'ONLOAD', 'TIME', renderTime);
+                }, 10);
+            };
+            // @see: http://stackoverflow.com/q/3763080/
+            if (window.attachEvent) {
+                window.attachEvent('onload', abOnLoadHandler);
+            } else if (window.addEventListener) {
+                window.addEventListener('load', abOnLoadHandler, false);
+            }
+        }
+    }
+
+    // Unleash
+    _gaWikiaPush(['send', 'pageview']);
+
+    /**
+     * Advertisement Tracker, pushed separatedly.
+     *
+     * To be used for all ad impression and click events
+     */
+    // Advertisment Account UA-32129071-1/UA-32129071-2
+    if (isProductionEnv) {
+        window.ga(
+            'create', 'UA-32129070-3', 'auto',
+            {
+                'name': 'ads',
+                'sampleRate': 100,
+                'allowLinker': true,
+                'userId': window.wgGAUserIdHash
+            }
+        );
+    } else {
+        window.ga(
+            'create', 'UA-32129070-3', 'auto',
+            {
+                'name': 'ads',
+                'sampleRate': 100,
+                'allowLinker': true,
+                'userId': window.wgGAUserIdHash
+            }
```

Comment:
Right. Added different accounts for prod and dev 6a371d630e2bf012b121e2d0a74b38c29c94f158